internal/fluentbit/config/builder/input.go

```diff
@@ -0,0 +1,71 @@
+package builder
+
+import (
+	"fmt"
+	"strings"
+
+	telemetryv1alpha1 "github.com/kyma-project/telemetry-manager/apis/telemetry/v1alpha1"
+	"github.com/kyma-project/telemetry-manager/internal/namespaces"
+)
+
+func createInputSection(pipeline *telemetryv1alpha1.LogPipeline, includePath, excludePath string) string {
+	inputBuilder := NewInputSectionBuilder()
+	inputBuilder.AddConfigParam("name", "tail")
+	inputBuilder.AddConfigParam("alias", pipeline.Name)
+	inputBuilder.AddConfigParam("path", includePath)
+	inputBuilder.AddConfigParam("exclude_path", excludePath)
+	inputBuilder.AddConfigParam("multiline.parser", "docker, cri, go, python, java")
+	inputBuilder.AddConfigParam("tag", fmt.Sprintf("%s.*", pipeline.Name))
+	inputBuilder.AddConfigParam("skip_long_lines", "on")
+	inputBuilder.AddConfigParam("db", fmt.Sprintf("/data/flb_%s.db", pipeline.Name))
+	inputBuilder.AddConfigParam("storage.type", "filesystem")
```

Comment:
Cant we use in memory `storage.type` ? Filesystem would store the logs on the node ? 
Also should we use `storage.pause_on_chunks_overlimit` ? https://docs.fluentbit.io/manual/administration/buffering-and-storage#input-section-configuration

Reply:
Keeping the filesystem storage for now was intended to keep the impact of this PR small. The decision to remove it is a follow-up. Would also do some tests with the `storage.pause_on_chunks_overlimit` as a follow-up, the behaviour is not entirely clear to me. The description sounds like it could also cause pausing before the buffer is filled up, what we do not want.